dali/test/python/test_operator_multipaste.py

```diff
@@ -148,7 +147,10 @@ def get_pipeline(
         if not out_anchor_top_left:
             kwargs["out_anchors"] = out_anchors
 
-        pasted = fn.multi_paste(resized, **kwargs)
+        if use_gpu:
+            kwargs["device"] = "gpu"
+
+        pasted = fn.multi_paste(resized.gpu() if use_gpu else resized, **kwargs)
```

Comment:
```suggestion
        pasted = fn.multi_paste(resized.gpu() if use_gpu else resized)
```

Reply:
Will remove kwargs["device"], but there are other kwargs that still have to be passed